misc/python/materialize/cloudtest/k8s/minio.py

```diff
@@ -13,10 +13,10 @@
 
 class Minio(K8sResource):
     def __init__(
-        self,
-        namespace: str = DEFAULT_K8S_NAMESPACE,
+        self, namespace: str = DEFAULT_K8S_NAMESPACE, apply_node_selectors: bool = False
```

Comment:
I think this needs to be True always and the selector should always be enabled. Do you forsee a case where minio is better allowed to float around the cluster?

Reply:
This will be necessary if I want to keep the original set of nodes for all other cloudtests. `apply_node_selectors` will only be true for node-recovery tests and false for other tests.

Reply:
In other words, it avoids that I will need dedicated workers with `supporting-services` label for the other cloudtests.